Controller/SubscriptionTypeController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+* This file is part of the Integrated package.
+*
+* (c) e-Active B.V. <integrated@e-active.nl>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Integrated\Bundle\SubscriptionBundle\Controller;
+
+use Integrated\Bundle\SubscriptionBundle\Entity\SubscriptionType;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @author Jacob de Graaf <jacob.de.graaf@windesheim.nl>
+ * @author Albert Bakker <albert-david.bakker@windesheim.nl>
+ */
+class SubscriptionTypeController extends Controller
```

Comment:
All controllers should be registered as a service, you can take a look at the website bundle -> Resources->config -> controllers.xml